packages/config/config/devices/0x0258/neo-ds07z.json

```
@@ -0,0 +1,158 @@
+{
+	"manufacturer": "Shenzhen Neo Electronics Co., Ltd.",
+	"manufacturerId": "0x0258",
+	"label": "NEO-DS07Z",
+	"description": "Door/Window Sensor Plus",
+	"devices": [
+		{
+			"productType": "0x0010",
+			"productId": "0x0702"
+		}
+	],
+	"firmwareVersion": {
+		"min": "2.0",
+		"max": "255.255"
+	},
+	"metadata": {
+		"inclusion": "1. Power Up The Device.\n2. Set Z-Wave Controller Into Inclusion Mode\n3. Press And Hold The Button For 5s Until White Led Lights Is On, Then Release The Button Before Led Turn Off.\nBlue Led Will Blink With 1s Interval Until Inclusion Successful.",
+		"exclusion": "1. Power Up The Device.\n2. Set Z-Wave Controller Into Exclusion Mode\n3. Press And Hold The Button For 5s Until White Led Lights Is On, Then Release The Button Before Led Turn Off.\nBlue Led Will Blink With 0.5sinterval Until Exclusion Successful.",
+		"reset": "1. Power Up The Device.\n2. Press And Hold The Button For 10s Until Pink Led Lights Is On, Then Release The Button Before Led Turn Off."
+	},
+	"paramInformation": [
+		{
+			"#": "1",
+			"label": "Led Indicated",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"unsigned": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 0
+				},
+				{
+					"label": "Enable",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "2",
+			"label": "Binary Sensor Report",
+			"description": "Enable SENSOR_BINARY_REPORT When Device Detects A Door/Window Opened Or Closed Event.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"unsigned": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 0
+				},
+				{
+					"label": "Enable",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "3",
+			"label": "Temperature Offset Value",
+			"description": "The Current Measuring Temperature Value Can Be Add And Minus A Value By This Setting. Temperature Offset Value = [Value] × 0.1 Degree Celsius / Fahrenheit (Us).Temperature Offset Value = [Value] × 0.1 Degree Celsius / Fahrenheit (Us).",
```

Description seems unnecessary. The relevant information about the unit should be converted into
```suggestion
			"unit": "0.1 (°C/°F)",
```
instead. Please do the same for the other similar parameters